### ISSUE TITLE
 "Time correction applied" flag lost during format conversion

### ISSUE THREAD

**Issue by nicoleroy-ipgp** [iris-edu/mseedconvert#3]
We are trying to convert to float32 all the old GEOSCOPE data encoded using "retired" Geoscope multiplexed format. I tried to use mseedconvert to do so but I am facing an issue with the "Time correction applied" bit

[G.AGD..VHE.D.1987.zip](https://github.com/iris-edu/mseedconvert/files/7897580/G.AGD.VHE.D.1987.zip)

I can convert this file to float32 using the following command:
`./mseedconvert -F 2 -E 4 -f -o G.AGD..VHE.D.1987.218.float32 G.AGD..VHE.D.1987.218 -vv`

But then the activity flag [Bit 1] "Time correction applied" is lost while the start time value does not seem to be affected, resulting in false timing:

```
> msi -n 1 -pp -z G.AGD..VHE.D.1987.218
Swapping multi-byte quantities in header
RECORD -- G_AGD__VHE_Q
        sequence number: '001252'
 data quality indicator: 'Q'
               reserved: ' '
           station code: 'AGD  '
            location ID: '  '
          channel codes: 'VHE'
           network code: 'G '
             start time: 1987,218,8:59:59.5391 (unused: 0)
      number of samples: 2016
     sample rate factor: 1  (0.1 samples per second)
 sample rate multiplier: -10
         activity flags: [01000000] 8 bits
                         [Bit 1] Time correction applied
    I/O and clock flags: [00000000] 8 bits
     data quality flags: [00000000] 8 bits
   number of blockettes: 1
        time correction: -4609
            data offset: 64
 first blockette offset: 48
          BLOCKETTE 1000: (Data Only SEED)
              next blockette: 0
                    encoding: GEOSCOPE Muxed 16/3 bit gain/exp (val:13)
                  byte order: Big endian (val:1)
               record length: 4096 (val:12)
               reserved byte: 0


> msi -n 1 -pp -z G.AGD..VHE.D.1987.218.float32
Swapping multi-byte quantities in header
RECORD -- G_AGD__VHE_D
        sequence number: '000000'
 data quality indicator: 'D'
               reserved: ' '
           station code: 'AGD  '
            location ID: '  '
          channel codes: 'VHE'
           network code: 'G '
             start time: 1987,218,8:59:59.5391 (unused: 0)
      number of samples: 1010
     sample rate factor: -10  (0.1 samples per second)
 sample rate multiplier: -1
         activity flags: [00000000] 8 bits
    I/O and clock flags: [00000000] 8 bits
     data quality flags: [00000000] 8 bits
   number of blockettes: 1
        time correction: -4609
            data offset: 56
 first blockette offset: 48
          BLOCKETTE 1000: (Data Only SEED)
              next blockette: 0
                    encoding: IEEE floating point (val:4)
                  byte order: Big endian (val:1)
               record length: 4096 (val:12)
               reserved byte: 0
```

**Comment by chad-iris** [comment]
Thanks @nicoleroy-ipgp for the excellent report and test data.

In the _draft_ miniSEED 3 format time corrections are always applied and this is the data model of the libmseed representation (and therefore this converter).  On quick look, the miniSEED 2 bit flag should be set whenever a non-zero time correction is included in the record, somewhere around here:
https://github.com/iris-edu/libmseed/blob/master/pack.c#L993

**Comment by chad-iris** [comment]
Fixed in v0.9.2.

`./mseedconvert -F 2 -E 4 -f -o G.AGD..VHE.D.1987.218.float32 G.AGD..VHE.D.1987.218 -vv`

now produces the following output data:

```
% msi -n 1 -pp -z G.AGD..VHE.D.1987.218.float32
Swapping multi-byte quantities in header
RECORD -- G_AGD__VHE_D
        sequence number: '000000'
 data quality indicator: 'D'
               reserved: ' '
           station code: 'AGD  '
            location ID: '  '
          channel codes: 'VHE'
           network code: 'G '
             start time: 1987,218,8:59:59.5391 (unused: 0)
      number of samples: 1010
     sample rate factor: -10  (0.1 samples per second)
 sample rate multiplier: -1
         activity flags: [01000000] 8 bits
                         [Bit 1] Time correction applied
    I/O and clock flags: [00000000] 8 bits
     data quality flags: [00000000] 8 bits
   number of blockettes: 1
        time correction: -4609
            data offset: 56
 first blockette offset: 48
          BLOCKETTE 1000: (Data Only SEED)
              next blockette: 0
                    encoding: IEEE floating point (val:4)
                  byte order: Big endian (val:1)
               record length: 4096 (val:12)
               reserved byte: 0
```
